File: onebot/src/main/java/cn/evole/onebot/sdk/event/message/MessageEvent.java
package cn.evole.onebot.sdk.event.message;

import cn.evole.onebot.sdk.event.Event;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;
import lombok.experimental.SuperBuilder;

/**
 * @author cnlimiter
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
@SuperBuilder(toBuilder = true)
@EqualsAndHashCode(callSuper = true)
public class MessageEvent extends Event {
    public String messageType;
    public long userId;
    public String message;
    public String rawMessage;
    public int font;
}

File: onebot/src/main/kotlin/sdk/event/message/MessageEvent.kt
package cn.evolvefield.onebot.sdk.event.message

import cn.evolvefield.onebot.sdk.event.Event

open class MessageEvent : Event() {
    var messageType = ""
    var userId = 0L
    var isJsonMessage = false
    var message = ""
    var rawMessage = ""
    var font = 0
}
